test/integration/scan_test.go

```diff
@@ -928,7 +929,8 @@ func copyResultsToTempDir() error {
 		return err
 	}
 	// Write data to dst
-	err = ioutil.WriteFile(realtime.ScaResolverResultsFileNameDir, data, 0644)
+	scaResolverResultsFileNameDir := filepath.Join(scaconfig.Params.WorkingDir(), realtime.ScaResolverResultsFileName)
+	err = ioutil.WriteFile(scaResolverResultsFileNameDir, data, 0644)
```

Comment:
suggestion: ioutil is deprecated, consider changing it to os.WriteFile
Note: There are several places in this pr that use ioutil